manifests/klusterlet/management/klusterlet-registration-clusterrole-addon-management.yaml

```diff
@@ -0,0 +1,12 @@
+# Permissions for registration agent to manage Hosted mode addons
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: open-cluster-management:management:{{ .KlusterletName }}-registration::addon-management
+rules:
+# registration agent needs secret permission for an arbitrary namespace to create hub-kubeconfig secret for an addon 
+# in Hosted mode
+- apiGroups: [""]
```

Comment:
is this the same as what role in managedcluster will apply? should we just use the same instead of copy another one?